engine/modelc/src/jni_util.h

```diff
@@ -28,6 +34,26 @@ namespace dmJNI
         }
     };
 
+
+#if defined(_WIN32)
+    LONG WINAPI ExceptionHandler(EXCEPTION_POINTERS* ptr);
+
+    #define DM_JNI_GUARD_SCOPE_BEGIN() \
+        __try { \
+
+    #define DM_JNI_GUARD_SCOPE_END(...) \
+        } __except (dmJNI::ExceptionHandler(GetExceptionInformation())) { \
+            __VA_ARGS__ \
+        }
```

Comment:
The MSVC intrinsics `__try`/`__except` managed to catch this access violation, whereas the previous code didn't.